bundles/config/org.eclipse.smarthome.config.core/src/main/java/org/eclipse/smarthome/config/core/ConfigDescriptionParameter.java

```diff
@@ -274,16 +322,46 @@ public String getDefault() {
     /**
      * Returns a human readable label for the configuration parameter.
      *
-     * @return a human readable label for the configuration parameter (could be null or empty)
+     * @return a human readable label for the configuration parameter (could be
+     *         null or empty)
      */
     public String getLabel() {
         return this.label;
     }
 
     /**
+     * Returns a the group for this configuration parameter.
+     *
+     * @return a group for the configuration parameter (could be null or empty)
```

Comment:
Also, in the XSD this is optional - again - not my doing so I'd probably prefer to take this outside of this PR.